desktop/conf.dist/hue.ini

```diff
@@ -1628,6 +1628,10 @@ submit_to=True
 # Optionally set this to control the caching strategy for files download
 ## file_download_cache_control=no-store, no-cache, max-age=0, must-revalidate, proxy-revalidate
 
+# A limit on a file size (bytes) that can be uploaded to a filesystem.
+# A value of -1 means there will be no limit.
+##upload_size_limit=-1
```

Comment:
What is the use case of setting `upload_size_limit=-1` rather than just not setting the limit at all (and have it default to `none` in pyton and `undefined` in JS)?

Reply:
I was following our code structure what we are doing with other limit cases so there -1 indicates no limit, hence I used that.